backend/events/apps.py

```diff
@@ -85,6 +89,24 @@ def on_tuples(cc_event, block_number, tx_id, tx_status):
                 queue=worker_queue
             )
 
+        if 'computePlan' in payload:
+            handle_compute_plan_events(payload['computePlan'])
```

Comment:
Should be outside of the for loop (on payload) and this should be ignored if `tx_status` is not valid.

Could be worth to handle the `tx_status` at the beginning of the function. It was done inside the for loop to easily log all skipped tuples.

Reply:
Good points, thank you!